README.md

```diff
@@ -25,3 +25,13 @@ This will enable pins O_3 and O_9 as PWM pins, which can be used with Viam's API
 ### ADC and DAC
 
 The [AIO Module](https://revolutionpi.com/en/tutorials/overview-aio) is used for analog inputs and outputs on the Revolution Pi. The module currently supports 4 analog readers and 2 analog writers. the RTD analog readers are currently not managed by this module. See [RTD Measurement Documentation](https://revolutionpi.com/en/tutorials/overview-aio/rtd-measurement) for the Revolution Pi for more information.
+
+### DoCommand
+
+A DoCommand is configured to read from any address supported in the Revolution Pi. The command is configured as
+
+```
+{"readAddress": <ADDRESS_NAME>}
```

Comment:
I know it wasnt part of this PR but the getStatus do command could also be added to readme

Reply:
I can add that, and I’ll tweak the command to be more representative of what it’s actually doing

Reply:
Ended up removing getStatus as a command, since the function is used as a part of the board initialization already.